### PR TITLE
Add elemental-seedimage-hooks package

### PR DESCRIPTION
**Pull request by frelon**
The elemental-seedimage-hooks package contains hooks to copy rpi firmware when building a raw disk image.

### CODE REVIEW

**Comment by frelon** [comment]
Recreated to see if the OBS integration will pick this up.

File: hooks/01_rpi-firmware.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+name: "Raspberry Pi post disk hook"
+stages:
+    after-disk:
+    - name: "Copy firmware to EFI partition"
```
Comment (davidcassany) [inline hooks/01_rpi-firmware.yaml:4]: Isn't this going to be executed in all disk builds? Raspberry and non raspberry, aarch64 and x86. If so probably we could think of some sort of if clause to limit the scope of this hook.

File: hooks/01_rpi-firmware.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+      commands:
+      - cp -r /iso/build/recovery.img.root/boot/vc/* /iso/build/efi/
```
Comment (davidcassany) [inline hooks/01_rpi-firmware.yaml:6]: I am not entirely sure which are those paths... I do not remember all the details of the build env now. But probably we could add some sort of clause depending on the existence of contents in /boot/vc/*. I assume those are firmware specific files.

**Review by davidcassany — APPROVED** [review]
Looks good to me, I just doubt about how to make explicit this hooks should only be executed in certain circumstances.

File: hooks/01_rpi-firmware.yaml (hunk 2 above)
Comment (kkaempf) [reply hooks/01_rpi-firmware.yaml:6]: See lines 104ff of [generate-rpi-image.sh](https://build.opensuse.org/package/view_file/isv:Rancher:Elemental:Dev/elemental-post-build-generate-rpi-image/generate-rpi-image.sh?expand=1)

* /boot/vc/* is the current (working, supported) firmware
* the [UEFI firmware](https://build.opensuse.org/package/show/isv:Rancher:Elemental:Dev/raspberrypi-firmware-uefi) is not fully working on Raspberry (yet ?!)

Comment (davidcassany) [reply hooks/01_rpi-firmware.yaml:6]: The probably we could add the same validation here.

Something like 

```yaml
    after-disk:
    - if: '[ -d "/iso/build/recovery.img.root/boot/vc" ]'
      name: "Copy firmware to EFI partition"
      commands:
      - cp -r /iso/build/recovery.img.root/boot/vc/* /iso/build/efi/
```

Comment (frelon) [reply hooks/01_rpi-firmware.yaml:6]: Yep good idea! We currently get an ugly error log on x86 in the seedimage build pod.. I will add the conditional!

**Pull request merged**